ts_ankle_draft.json

```diff
@@ -0,0 +1,42 @@
+{
+  "information": {
+    "summary": "This file contains validation data for the ankle_draft() function in pythermalcomfort,jsthermalconfort,calcAD() in R, ",
+    "version": "1.0.0",
+    "license": "MIT"
+  },
+  "tolerance": {
+    "ppd": 0.1
+  },
+  "data": [
+    {
+      "inputs": {
+        "tdb": 25,
+        "tr": 25,
+        "v": 0.2,
+        "rh": 50,
+        "met": 1.2,
+        "clo": 0.5,
+        "v_ankle": 0.3,
+        "units": "SI"
+      },
+      "outputs": {
+        "PPD_ad": 18.5
```

Comment:
this should be lower case

Reply:
I noticed that this is an error in pythermalcomfotr and we need to fix it. I have opened an issue.

Reply:
Sure！Thank you.

Reply:
@Wangjiayi1115 in future no need to say thank you, on the other hand please let me know how and where you fixed the issue